l10n_it_fatturapa_in_notifications/account.py

```diff
@@ -19,11 +19,37 @@
 ##############################################################################
 
 from openerp.osv import fields, orm
+import netsvc
 
 
 class AccountInvoice(orm.Model):
     _inherit = "account.invoice"
     _columns = {
         'result_notification_id': fields.many2one(
-            'fatturapa.notification', "Result notification", readonly=True)
+            'fatturapa.notification', "Result notification", readonly=True),
+        'rejected': fields.boolean('Rejected')
         }
+
+    _defaults = {
+        'rejected': False
+    }
+
+    def action_cancel_draft(self, cr, uid, ids, *args):
+        if not hasattr(ids, '__iter__'):
+            ids = [ids]
+        invoice = self.browse(cr, uid, ids[0])
```

Comment:
In questo modo funziona solo per la prima fattura di ids.
Invece dovresti ciclare su tutte le fatture e, per ognuna, se rejected, sollevare eccezione